### PR TITLE
Fix tracing leak from working with james.

### PR DESCRIPTION
**Pull request by pschuh**
Two problems:
- When returning a list of tensors, the CTensorHandles represent a reference. These need to be passed owning.
- TFE_Ops will leak tensors if not freed.

### CODE REVIEW

**Review by bgogul — APPROVED** [review]

**Comment by bgogul** [comment]
Nice!

**Comment by pschuh** [comment]
@swift-ci please test TensorFlow

**Comment by pschuh** [comment]
@swift-ci please test TensorFlow gpu

File: stdlib/public/TensorFlow/CompilerRuntime.swift [hunk 1]
```diff
@@ -829,6 +830,20 @@ private extension TensorGroup {
     TF_DeleteStatus(status)
     self.init(_owning: buffer)
   }
+
+  init<C: Collection>(_owning input: C) where C.Element == CTensorHandle {
```
Comment (rxwei) [inline stdlib/public/TensorFlow/CompilerRuntime.swift]:
```suggestion
  init<C : Collection>(_owning input: C) where C.Element == CTensorHandle {
```

**Review by rxwei — COMMENTED** [review]
yay!

**Comment by pschuh** [comment]
@swift-ci please test TensorFlow gpu

**Comment by pschuh** [comment]
GPU failure seems unrelated (tensor slicing)

**Pull request merged**